zmon_aws_agent/aws.py

```diff
@@ -748,6 +748,11 @@ def get_limits(region, acc, apps, elbs):
         'elb-max-count': 20,
         'elb-used-count': len(elbs),
     }
+    for e in entities:
+        if e.get('type') == 'aws_limits':
+            limits.update(e)
+            break
```

Comment:
We need to move setting ``ec2-used-instances``, ``elb-used-count`` and ``ec2-used-spot-instances`` after this update since we need the most recent values.